spec/Denormalizer/CompositeDenormalizerSpec.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace spec\SyliusLabs\RabbitMqSimpleBusBundle\Denormalizer;
+
+use PhpAmqpLib\Message\AMQPMessage;
+use PhpSpec\ObjectBehavior;
+use SyliusLabs\RabbitMqSimpleBusBundle\Denormalizer\DenormalizationFailedException;
+use SyliusLabs\RabbitMqSimpleBusBundle\Denormalizer\DenormalizerInterface;
+
+/**
+ * @author Kamil Kokot <kamil@kokot.me>
+ */
+final class CompositeDenormalizerSpec extends ObjectBehavior
+{
+    function it_is_a_denormalizer()
+    {
+        $this->shouldImplement(DenormalizerInterface::class);
+    }
+
+    function it_does_not_support_a_message_if_there_are_no_denormalizers()
+    {
+        $amqpMessage = new AMQPMessage('Message body');
+
+        $this->supports($amqpMessage)->shouldReturn(false);
+        $this->shouldThrow(DenormalizationFailedException::class)->during('denormalize', [$amqpMessage]);
```

Comment:
Are you sure that it should throw `DenormalizationFailedException` when there are no denormalizers registered?